packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -498,6 +498,12 @@ export class LoggingError extends Error {
         props?: ITelemetryProperties,
     ) {
         super(message);
+        if (props) {
```

Comment:
unnecessary 🤷‍♂️

Reply:
I prefer stronger typing (changing addTelemetryProperties to not accept undefined).